crates/compiler/builtins/roc/Dict.roc

```diff
@@ -206,6 +225,22 @@ walk = \@Dict { data }, initialState, transform ->
 ##
 ## As such, it is typically better for performance to use this over [Dict.walk]
 ## if returning `Break` earlier than the last element is expected to be common.
+## Below is a real world example
+## ```
+## transform = \count, k, v ->
+##     if k == "Orange" then
+##         Break count
+##     else
+##         Continue (count + v)
+##
+## foodDict = Dict.empty {}
+##     |> Dict.insert "Apples" 12
+##     |> Dict.insert "Orange" 24
+##     |> Dict.insert "Banana" 36
+##
+## walkUntil = Dict.walkUntil foodDict 0 transform
+## expect walkUntil == Bool.isEq 12
```

Comment:
I think this might be a typo, consider using the infix `==`?

```
The isEq function expects 2 arguments, but it got only 1:

23│  expect walkUntil == Bool.isEq 12
```